### PR TITLE
Update FieldsListPlugin.php

### PR DESCRIPTION
**Pull request by varaprasad16**
Pull Request for Issue #37429 

### Summary of Changes
Written the for loop code for copying data from params to $data after cloning the params data. so that $data will copy the data this->params data

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request

image

### CODE REVIEW

**Comment by richard67** [comment]
@varaprasad16 Your PR has code style errors. You can see that when checking the details link of the drone CI check at the bottom of the PR and then selecting the failed test on the page: https://ci.joomla.org/joomla/joomla-cms/53398/1/6 . Could you fix that? Thanks in advance.

**Comment by richard67** [comment]
@varaprasad16 Now there are other code style errors: https://ci.joomla.org/joomla/joomla-cms/53403/1/6 . Maybe you should adjust your editor so it shows you spaces and tabs.

**Comment by richard67** [comment]
@varaprasad16 Your last commit makes it even worse by adding another tab to an empty line.

File: administrator/components/com_fields/src/Plugin/FieldsListPlugin.php [hunk 1]
```diff
@@ -70,8 +70,15 @@ public function getOptionsFromField($field)
 
 		// Fetch the options from the plugin
 		$params = clone $this->params;
+		
+		foreach ($params->get('options', array()) as $option)
+		{
+			$op = (object) $option;
+			$data[$op->value] = $op->name;
+		}
+
```
Comment (richard67) [inline administrator/components/com_fields/src/Plugin/FieldsListPlugin.php]:
```suggestion

		foreach ($params->get('options', array()) as $option)
		{
			$op = (object) $option;
			$data[$op->value] = $op->name;
		}

```
Empty lines should not contain spaces or tabs.

Comment (richard67) [reply administrator/components/com_fields/src/Plugin/FieldsListPlugin.php]: Same for line 81 below.

Comment (varaprasad16) [reply administrator/components/com_fields/src/Plugin/FieldsListPlugin.php]: Yes, sir, I've made changes according to your suggestion.

**Comment by richard67** [comment]
Currently it's not clear yet if the issue solved by this PR is really an issue or not, see my comment in the issue: https://github.com/joomla/joomla-cms/issues/37429#issuecomment-1101297300 .

**Comment by richard67** [comment]
@varaprasad16 Thank you for your first pull request (PR) for the Joomla CMS. Unfortunately it has turned out that the issue which your PR would solve is not an issue but expected behaviour. See the comments there. So I'll close this PR now. I hope you are not disappointed. This can happen sometimes, also to experienced contributors. Don't give up, the next one will be more lucky.

**Pull request closed without merging**